scripts/building/dev_requirements.txt

```diff
@@ -9,3 +9,8 @@ pytest-sugar
 pytest-timeout
 azure-keyvault
 azure-identity
+# record and replay http requests for pfazure tests
```

Comment:
can we use `src/promptflow/dev_requirements.txt`? otherwise we might need double updates for dev reqs.

Reply:
Think we can use that, that's the dev requirement file what my original comment implied.

Reply:
Why need this duplicate since `src/promptflow/dev_requirements.txt` already has the deps

Reply:
@crazygao ping on this

Reply:
will unify them in future pr.